### PR TITLE
windows: Fix path for file sanitization

### PR DESCRIPTION
**Pull request by walac**

### CODE REVIEW

File: src/tests/eina/eina_test_file.c [hunk 1]
```diff
@@ -527,7 +527,11 @@ static const struct {
    const char *test;
    const char *result;
 } sanitize[] = {
+#if defined _WIN32
+  { "C:/home/mydir/../myfile", "C:/home/myfile" }
```
Comment (joaoantoniocardoso) [inline src/tests/eina/eina_test_file.c:531]: Wouldn't be better to use `\\` instead of `/`?

Comment (Coquinho) [reply src/tests/eina/eina_test_file.c:531]: oh, good point, this change makes the `eina_test_file_path` pass through.

Comment (Coquinho) [reply src/tests/eina/eina_test_file.c:531]: @walac do you know why using `/` is working?

Comment (walac) [reply src/tests/eina/eina_test_file.c:531]: Windows supports `/` as a path separator. I am working on more test cases for this.

Comment (joaoantoniocardoso) [reply src/tests/eina/eina_test_file.c:531]: Indeed but I don't see as an option of choice for win32 api as its functions will only return `\\` as path separators.

Comment (walac) [reply src/tests/eina/eina_test_file.c]: I am not sure I am following. In the updated patch I added tests for the `\` seperator cases.

Comment (joaoantoniocardoso) [reply src/tests/eina/eina_test_file.c]: yeah, the updated is ok to me.

**Pull request merged**